### ISSUE TITLE
MS-DIAL command line

### ISSUE THREAD

**Issue by NitTza** [systemsomicslab/MsdialWorkbench#125]
Hi, 
I am asking for a guidance to operate MS-DIAL ver 4 using a command line in a python code
Thanks, 
Nitzan

**Comment by kozo2** [comment]
Hi @NitTza 

Currently MS-DIAL does not have any API that can be used from Python,
but there is command line version of MS-DIAL4.
http://prime.psc.riken.jp/compms/msdial/consoleapp.html

(The consoleapp for MS-DIAL5 is not created yet.
https://github.com/systemsomicslab/MsdialWorkbench/issues/131)

The command line tool simply takes raw data and parameters as input and outputs files for detected peaks  and the alignment.
In other words, it is just files in and files out.

So I would suggest using a combination of Snakemake and http://prime.psc.riken.jp/compms/msdial/consoleapp.html .

You can write Python in the Snakemake workflow (==Snakefile).
